tools/rtpsrelay/utility.h

```diff
@@ -24,6 +24,50 @@ inline std::string guid_to_string(const OpenDDS::DCPS::GUID_t& a_guid)
   return ss.str();
 }
 
+struct RelayAddressesLessThan {
+  inline bool operator()(const RtpsRelay::RelayAddresses& x,
+                         const RtpsRelay::RelayAddresses& y) const
+  {
+    if (x.spdp_relay_address() != y.spdp_relay_address()) {
+      return x.spdp_relay_address() < y.spdp_relay_address();
+    }
+    if (x.sedp_relay_address() != y.sedp_relay_address()) {
+      return x.sedp_relay_address() < y.sedp_relay_address();
+    }
+    return x.data_relay_address() < y.data_relay_address();
+  }
+};
+
+struct GuidAddr {
+  OpenDDS::DCPS::RepoId guid;
```

Comment:
`#include` the header file for `RepoId`, `GUID_UNKNOWN`, `GUID_tKeyLessThan`, and the `operator==`/`operator!=` overloads